components/develop/modules/ROOT/nav.adoc

```diff
@@ -5,6 +5,7 @@
 ** xref:hubs/community.adoc[Community]
 ** xref:hubs/defi.adoc[DeFI]
 ** xref:hubs/gaming.adoc[Gaming]
+*** xref:hubs/integration-steps/gaming-steps.adoc[Integration Steps]
```

Comment:
Alternative layout - 

Change integration-steps directory to gaming directory and change gaming-steps.adoc -> integration-steps.adoc

Then files can be grouped by chain type/industry type